target/linux/ramips/image/mt76x8.mk

```diff
@@ -342,6 +342,39 @@ define Device/kroks_kndrt31r19
 endef
 TARGET_DEVICES += kroks_kndrt31r19
 
+define Device/linksys_ac1200
+  IMAGE_SIZE := 16000k
+  DEVICE_VENDOR := Linksys
+  DEVICE_PACKAGES := kmod-mt76x2
+endef
```

Comment:
As all these boards are the same I would prefer the ALT variant:
````
define Device/linksys_e2500-v4
  IMAGE_SIZE := 16000k
  DEVICE_VENDOR := Linksys
  DEVICE_MODEL := E2500
  DEVICE_VARIANT := v4
  DEVICE_ALT0_VENDOR := Linksys
  DEVICE_ALT0_MODEL := E5300
  DEVICE_ALT1_VENDOR := Linksys
  DEVICE_ALT1_MODEL := E5350
  DEVICE_ALT2_VENDOR := Linksys
  DEVICE_ALT2_MODEL := E5400
  DEVICE_PACKAGES := kmod-mt76x2
endef
TARGET_DEVICES += linksys_e2500-v4
````

Then you need only one dts file.